tests/prefetch_related/test_uuid.py

```diff
@@ -52,6 +53,14 @@ def test_prefetch_related_from_uuid_model_to_uuid_model_with_values_flat(self):
             [pet.id],
         )
 
+    def test_prefetch_related_objects_str_as_uuid_fk(self):
+        pet = Pet.objects.create(name="Fifi")
+        toy_ball = Toy(pet_id=pet.id, name="Ball")
+        toy_bone = Toy(pet_id=str(pet.id), name="Bone")
+        prefetch_related_objects([toy_ball, toy_bone], "pet")
+        self.assertEqual(pet, toy_ball.pet)
+        self.assertEqual(pet, toy_bone.pet)
```

Comment:
```suggestion
        with self.assertNumQueries(0):
            self.assertEqual(pet, toy_ball.pet)
            self.assertEqual(pet, toy_bone.pet)
```

Reply:
That makes perfect sense. Thanks, applied the suggestion.